src/patternfly/components/Tabs/tabs.scss

```diff
@@ -220,6 +240,33 @@
     border: var(--pf-c-tabs__item--BorderWidth) solid var(--pf-c-tabs__item--BorderColor);
   }
 
+  &.pf-m-secondary {
+    &::before {
+      border: 0;
+    }
+
+    &.pf-m-hover,
+    &:hover,
+    &.pf-m-active,
+    &:active,
+    &.pf-m-focus,
+    &:focus {
+      color: var(--pf-c-tabs__scroll-button--m-secondary--hover--Color);
+
+      &::after {
```

Comment:
This generates a pretty long selector just to disable this `border-top`

```
.pf-c-tabs__scroll-button.pf-m-secondary.pf-m-hover::after, .pf-c-tabs__scroll-button.pf-m-secondary:hover::after, .pf-c-tabs__scroll-button.pf-m-secondary.pf-m-active::after, .pf-c-tabs__scroll-button.pf-m-secondary:active::after, .pf-c-tabs__scroll-button.pf-m-secondary.pf-m-focus::after, .pf-c-tabs__scroll-button.pf-m-secondary:focus::after {
    border-top: 0;
}
```

Can you just not define it on `.pf-m-secondary`?